exchanges/binance/binance_wrapper.go

```diff
@@ -124,8 +127,35 @@ func (b *Binance) UpdateOrderbook(p pair.CurrencyPair, assetType string) (orderb
 // GetAccountInfo retrieves balances for all enabled currencies for the
 // Bithumb exchange
 func (b *Binance) GetAccountInfo() (exchange.AccountInfo, error) {
-	var response exchange.AccountInfo
-	return response, common.ErrNotYetImplemented
+	var info exchange.AccountInfo
+	raw, err := b.GetAccount()
+	if err != nil {
+		return info, err
+	}
+
+	var currencyBalance []exchange.AccountCurrencyInfo
+	for _, balance := range raw.Balances {
+		freeCurrency, err := strconv.ParseFloat(balance.Free, 64)
+		if err != nil {
```

Comment:
If the first currency has an error (for some theoretical reason) but the rest are fine, then this will cut off retrieving the details for the other currencies. I'm not sure what to do here, but thought I'd highlight it as a potential issue.

You could possibly add an error string to exchange.AccountCurrencyInfo, so if there is an issue with any singular currency for any reason, then it can be stored there

Reply:
This will almost be impossible to error unless they start sending "uiweuhasifdg0.0001" for balance.Free which I think will be some bigger issues on the exchange.